docs/en/core/defaults-files.md

```diff
@@ -81,17 +81,60 @@ variables:
 You can also make use of this templating system yourself. If you want to write a custom template, you can add statements such as `$if(myvariable)$$myvariable$$endif$` and insert the variable `myvariable` into any frontmatter or defaults file, and it will be replaced with whatever value you set it to.
 
 
-### Microsoft Word Template (.docx)
+### Reference Documents (.docx, .odt, .pptx) 
 
-For export as a Word document, you need a .docx document that serves as a template. This file should be stored in the same folder as your markdown files and added to the Word export Defaults file in Assets Manager. 
+For export as Microsoft Word document, OpenDocument, or PowerPoint presentation, Pandoc uses a reference document that serves as a style template. This section is written in relation to Word docx, but the same should apply for either ODT or PPTX. 
 
-#### Reference Document 
+#### Producing the Reference Document
 
-Use an existing Word document or download the reference.docx from Pandoc and change the styles. Your reference doc is a Word file that you format to your preferences. You need to adjust the Word Styles (Paragraph, Heading, Body Text, Footnote Text, etc) for it to apply. 
+For best results, it is recommended to download the reference document directly from Pandoc. It should already be installed with Zettlr, but if not, see [Pandoc installation](https://docs.zettlr.com/en/installing-pandoc/) instructions. 
+
+##### Option 1: Using Pandoc from Command line or Terminal 
```

Comment:
Since option 2 is definitely more cumbersome, I would suggest to remove that option and only leave this option.